src/evaluate/evaluator/question_answering.py

```diff
@@ -61,13 +61,24 @@ def prepare_data(
         context_column: str,
         id_column: str,
         label_column: str,
+        squad_v2_format: bool,
+        data_split: str = None,
     ):
         """Prepare data."""
+        if isinstance(data, str):
+            data_split = self.get_dataset_split(data, data_split)
+            data = load_dataset(data, split=data_split)
+
+        if squad_v2_format is None:
```

Comment:
This has to be moved because it acts on the `data` object, which isn't passed around outside of `prepare_data`.